index.html

```diff
@@ -1642,6 +1648,43 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <dfn>forget()</dfn> method
+        </h3>
+        <div class="note">
+          <p>
+            The user agent MAY decide to combine permissions across APIs, for
+            instance tracking WebHID + WebUSB device access under a unified
+            low-level device access permission. This is why this method may
+            also revoke additional (unspecified yet) permissions in the future.
+          </p>
+        </div>
+        <p>
+          The {{HIDDevice/forget()}} method steps are:
+        </p>
+        <ol>
+          <li>Let |promise:Promise| be [=a new promise=].
+          </li>
+          <li>Set [=this=].{{HIDDevice/[[state]]}} to `"forgetting"`.
```

Comment:
The device might be open or closed. If it's open, I think we should close it (and issue relevant callbacks) as part of these steps.

Reply:
By closing it when forgetting it, we're introducing race conditions in the spec as `{{HIDDevice/[[state]]}}` would become `"closing"`, not `"forgetting"` anymore.
What would be the best way to handle this then?

Reply:
I think this is fine now that we invoke pending callbacks on forgetting.